Extension/ThirdPartyNotices.txt

```diff
@@ -375,7 +425,7 @@ SOFTWARE.
 
 -------------------------------------------------------------------
 
-vscode-cpptools 2.1.1 - MIT
+vscode-cpptools 2.1.2 - MIT

```

Comment:
should this be vscode-cpptools-api?

Reply:
It's called "vscode-cpptools" in the package.json.